backend/schemas/event.ts

```diff
@@ -1,53 +1,56 @@
 const event = {
     name: "event",
-    title: "Events",
+    title: "Event",
     type: "document",
     fields: [
         {
             name: "title",
-            title: "Title",
+            title: "Title på Event ",
+            description:"Tittel til event. F.eks 'PR VERV'",
             type: "string",
```

Comment:
Recommend using another event like "Casebreaker". 

(PR verv was not an event, but something the head of PR added through a mistiake)